### PR TITLE
Create initial release for blockhead broker

### PR DESCRIPTION
**Pull request by swetharepakula**
- includes a sample manifest to deploy to bosh-lite
  - cloudfoundry-incubator/blockhead/issues/4

Signed-off-by: Morgan Bauer <mbauer@us.ibm.com>

### CODE REVIEW

File: blockhead.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+name: blockhead
+
+addons:
+- name: bpm
+  jobs:
+  - name: bpm
+    release: bpm
+
+instance_groups:
+- name: blockhead-broker
```
Comment (nimakaviani) [inline blockhead.yml]: let's rename this to `broker`. the deployment name is good enough for the qualifier

**Review by nimakaviani — COMMENTED** [review]
just a quick rename of the broker job. otherwise looks good.

**Comment by nimakaviani** [comment]
successfully deployed to bosh-lite and it successfully started the broker. 

looks good.

**Comment by MHBauer** [comment]
@nimakaviani does that look good? Do you need to try it again or is this a safe change?

**Pull request merged**